### ISSUE TITLE
Use source amount for creating charges via webhook

### ISSUE THREAD

**Issue by JanRogge** [pickware/ShopwareStripePayment#111]
Currently we try to get the amount out of the order when we try to create a charge via webhook. But our logic for net orders is wrong which causes charges to fail because the amount of the inital source and the charge are not the same.

Since the amount should not change between the source and charge anyway since it then fails on stripes site we should just use the amount that the source gives us. This is already the case in the newer version for Shopware 6.

**Comment by maximiliankern** [comment]
During the test, I ran into the following error (see screenshot). After I created a net order (B2B AT) and selected Klarna as the payment method, I was not redirected to Klarna and the following error message appeared. The error occurs both before the fix and after the fix. I am not 100% sure if this error is related to the original problem. 

<img width="1142" alt="Bildschirmfoto 2022-01-27 um 11 03 12" src="https://user-images.githubusercontent.com/61687007/151341512-359fe503-cd03-4aa4-9d08-d1f45f0d9c03.png">
